tools/appsre-build-worker-packer.sh

```diff
@@ -209,4 +224,7 @@ $CONTAINER_RUNTIME run --rm \
                    -e PKR_VAR_composer_commit="$COMMIT_SHA" \
                    -e PKR_VAR_ansible_skip_tags="$SKIP_TAGS" \
                    -e PKR_VAR_skip_create_ami="$SKIP_CREATE_AMI" \
+                   -e ARCH="$ARCH" \
```

Comment:
We need either a default here, or change the jenkins job to set this before we merge.